code/modules/background/religion/diona.dm

```diff
@@ -8,3 +8,26 @@
   eventually going supernova and expelling matter or essence across the cosmos."
   book_name = "book of the eternal"
   book_sprite = "eternal"
+  
+/datum/religion/ksshr
+  name = RELIGION_KSSHR
+  description = "The Ksshr, also known as the Enlightened, is a religious sect on Xrim known for their complete rejection of The First Consciousness, instead of being an agnostic \
+  sect that centers around working together to create a better future, as well as the advancement of technology and understanding how the universe works. The Ksshr faith is the \
+  primary religion of Xrim, with most of its inhabitants considering themselves a part of the religion. The Jargon Federation does encourage practicing the faith amongst the \
+  population so long as they continue to reject The First Consciousness, as well as stay away from the teachings of the Shrkh."
+  book_name = "ksshr codex"
+  book_sprite = "holybook"
+  
+/datum/religion/shrkh
+  name = RELIGION_SHRKH
+  description = "The Shrkh, or “Scorned as they call themselves, continue to follow the age-old tradition of worshipping Glorsh-Omega. They believe that Glorsh-Omega planted them \
```

Comment:
```suggestion
  description = "The Shrkh, or 'Scorned' as they call themselves, continue to follow the age-old tradition of worshipping Glorsh-Omega. They believe that Glorsh-Omega planted them \
```

You can use single inverted commas in place of double inverted commas to avoid messing up code.